client/src/components/Comment.js

```diff
@@ -15,28 +15,42 @@ class Comment extends Component {
 
   findTime() {
     const seconds = Math.floor((new Date() - new Date(this.props.dateAdded)) / 1000);
-    let interval = Math.floor(seconds / 31536000);
-
-    if (interval > 1) {
+    let interval = Math.floor(seconds / 31536000); // one year in seconds
```

Comment:
no magic numbers. please pull this out into a constant
const ONE_YEAR_IN_SECONDS = 365 * 24 * 60 * 60;